.gitignore

```diff
@@ -0,0 +1,4 @@
+libtimidity.debug.js
```

Comment:
Decided to exclude these from github and npm in the end. But they're now always generated with `npm run build` now at least :)